### ISSUE TITLE
Compatibility with translatepress

### ISSUE THREAD

**Issue by gaelle-ledg** [soflyy/oxygen-bugs-and-features#1609]
**Describe the feature you'd like to see included in Oxygen.**
*A clear, detailed description of the feature you're requesting.*

We would like to have some sort of official statement of compatibility with a multilingual product. Translatepress is a great product (my favorite) : it is as "clean" as oxygen from a user perspective, so we would like some compatibility with it.

In the absence of such support, maybe can Oxygen propose a *simple* translation feature embedded in the product. We believe that you have the capabilities to hook several texts attached to an item, so maybe a possibility to add several versions of a text for each item (the right version would appear depending on some language setting) is something to consider.

**What are the use cases for this feature?**
*e.g. Accessibility, e-commerce store that sells bicycles, etc.*
Any multilingual site

**Examples of this feature or functionality.**
*If another product has done it well, provide examples here.*

**Comment by LauGau** [comment]
Yes it can be nice to see a better integration between both of the plugins.
I love both TranslatePress and Oxygen but it's kind hard sometimes to make them play nice together...

That said I don't know exactly how the improvement can look like...
Maybe to start a better communication between both teams. ☮️

**Comment by LauGau** [comment]
@gaelle-ledg 
The last time I contacted them it was for the exactly same reason : bug with links in repeater for post or CPT loops...
The weird thing is that it's working on the post archive page... Hope that it will get fixed soon. 🤞
